<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>921</width>
     <height>581</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Database viewer</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="reloadButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>reload_icon.png</normaloff>reload_icon.png</iconset>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="undoButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>undo_icon.png</normaloff>undo_icon.png</iconset>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="redoButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>redo_icon.png</normaloff>redo_icon.png</iconset>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
